src/compiler/crystal/syntax/to_s.cr

```diff
@@ -1414,12 +1414,11 @@ module Crystal
           printed_arg = true
         end
         if named_args = node.named_args
-          @str << ", " if printed_arg
-          named_args.each do |named_arg|
+          named_args.each_with_index do |named_arg, i|
+            @str << ", " if printed_arg || i > 0
```

Comment:
The variable `i` is not needed here. And `printed_arg = true` below can remain. That's the only change needed (you only needed to move exactly one line one position down ^_^)